scripts/verify/create-staking-pool.ts

```diff
@@ -0,0 +1,72 @@
+import {ethers} from 'hardhat'
+import {StakingPoolMediator} from '../../typechain-types'
+import {log} from '../../config/logging'
+import {
+    addressEnvironmentVariable,
+    bigintEnvironmentVariable
+} from '../utils/environment-variable'
+import {logCreateStakingPoolEvents} from '../utils/transaction-event-log'
+import {getTimestampNow} from '../../test/framework/time'
+import {RewardType} from '../../test/event/staking/staking-events'
+
+async function createManagedStakingPool(
+    mediatorAddress: string,
+    creatorAddress: string,
+    daoId: bigint,
+    collateralTokensAddress: string
+): Promise<void> {
+    const mediatorFactory = await ethers.getContractFactory(
+        'StakingPoolMediator'
+    )
+    const mediator = <StakingPoolMediator>(
+        mediatorFactory.attach(mediatorAddress)
+    )
+
+    const creatorFactory = await ethers.getContractFactory('StakingPoolFactory')
+    const creator = <StakingPoolMediator>creatorFactory.attach(creatorAddress)
+
+    log.info('Creating a new managed staking pool..')
+
+    // how should we populate this with sensible test values?
```

Comment:
the bond example just uses dummy vals

Reply:
The bonds verify, doesn't automatically validate the details (potential for an improvement there), instead leaving as a manual step.

I'm fine for the staking to do the same. Original intent of the verify was to to check everything has been deployed & configured correctly i.e. we have a installed system rather than we have a logically correct system (rely on UT/IT/AT for that).